data/hai/unfettered 0 prologue.txt

```diff
@@ -943,7 +951,7 @@ mission "<uhai> 2: Prove Your Strength"
 			label fire
 			`	In the background you hear fires roaring, it seems the damage to his ship was quite serious indeed.`
 			choice
-				`	"Be careful with provoking me. You may have gotten killed. Do not count on me to repair your ship."`
+				`	"Be careful with provoking men you may have gotten killed! And don't count on me to repair your ship."`
```

Comment:
> Be careful with provoking men

1. We don't specify FL's gender
2. ~~why~~

Reply:
bruh this was not intended